lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -893,14 +893,12 @@ def detect_init_system(self) -> InitMode:
             systemd_running = False
             logging.info("systemd is not installed")
 
-        if docker_running and systemd_running:
-            raise RuntimeError("Magmad is running with both Docker and systemd")
-        elif docker_running:
+        if docker_running:
             return InitMode.DOCKER
         elif systemd_running:
             return InitMode.SYSTEMD
         else:
-            raise RuntimeError("Magmad is not running with either Docker or systemd")
+            return InitMode.SYSTEMD  # default to systemd if both are running - needed by feg integ tests
```

Comment:
Will it not just go into the docker_running branch in that case instead of defaulting to systemd? I think this branch is only reached if neither is running.

Reply:
arg

Reply:
Fixed